docs/swagger/chain-kujira-routes.yml

```diff
@@ -0,0 +1,1165 @@
+paths:
+  /chain/kujira/:
```

Comment:
I don't agree that we should add these Kujira specific routes. The point of https://github.com/hummingbot/gateway/pull/147 was to standardize common interactions with various chains under a single set of routes.

These routes below introduce different patterns for interactions such as fetching tokens that are already handled by the standard routes. This Kujira connector should fit into the existing model, and changes to the model should be proposed via PRP.

Reply:
We will make changes to do not expose the specific routes.

Reply:
We only make one request though, our current test coverage is based on that and we have 81% coverage now.
Because we have much more functionality (and we would like to maintain a separate fork then) we would need to do a too much big refactor if we try to remove everything that is apart from the current CLOB routes.
So what we propose is basically change the code so no one from the Client and the Gateway would be able to use the specific Kujira routes (and we will remove all swagger, json responses, and other related things from here) but if they want they could then use our fork which would provide some extra routes. Would that be ok @fengtality?